source/jormungandr/jormungandr/default_settings.py

```diff
@@ -109,3 +109,6 @@
     
 GRAPHICAL_ISOCHRONE = False
 HEAT_MAP = False
+# This parameter are used to apply gevent's monkey patch
+# The Goal is to activate parallel calling valhalla, without the patch, parallel http calling may not work
+PATCH_WITH_GEVENT_SOCKET = False
```

Comment:
If it is the code used on prod, it should be the one mainly tested... If we can make it crash on artemis, that's nicer than on prod :)